mono/utils/mono-threads.h

```diff
@@ -200,9 +200,13 @@ typedef struct _MonoThreadInfo {
 	/*Tells if this thread was created by the runtime or not.*/
 	gboolean runtime_thread;
 
-	/* Max stack bounds, all valid addresses must be between [stack_start_limit, stack_end[ */
+	/* Max stack bounds, all valid addresses must be between [stack_start_limit, stack_end) */
 	void *stack_start_limit, *stack_end;
 
+#if HOST_WIN32
+	struct _TEB* windows_teb;
```

Comment:
If TEB is private and TIB is public, shouldn't we store TIB here instead, since we are casting it to a TIB when used?

Reply:
Maybe put it together with the other Windows specific members under USE_WINDOWS_BACKEND?

Reply:
I don’t see the point of that define. Is It because there are pthreads libraries for Windows?? That would make it wrong. But I don’t know what it means.

Reply:
Oh you mean struct arrangement. Sure.

Reply:
I thought you meant different ifdef.

Reply:
It includes things that are windows specific and is set when HOST_WIN32 so instead of having two windows defines in this struct, I was thinking that we could just add it bellow the existing one USE_WINDOWS_BACKEND that is set when we HOST_WIN32 is set.

Reply:
Yes I think I understand and will do (didn't look at the code yet).